LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Abhilash Kishore
```

Comment:
Can you change this please to `Copyright (c) 2022 Abhilash Kishore, Mathew Teoh`? 

(Following multi-person license guidelines I found [here](https://law.stackexchange.com/questions/16847/how-should-multiple-copyright-holders-be-credited-using-the-mit-license))

If we get to a point in the future where it's too cumbersome to list every contributor like this, I'll think of something else.